src/components/Table/utils/index.ts

```diff
@@ -0,0 +1,14 @@
+export * from "./columns";
+export { createRowLookup } from "./GridRowLookup";
```

Comment:
Thought: Wdyt about skipping the barrel export for these sub-directories like components/utils/etc, and have importers just go directly to the file?

I think the barrel file makes since for `Table/index.ts` where we're controlling our public API, but :shrug: otherwise/internally it seems like more exports to forget adding new symbols to :-)

Reply:
Sure sounds good to me.